core/reducers/index.js

```diff
@@ -14,7 +15,8 @@ const rootReducer = combineReducers({
   modals,
   sort,
   filter,
-  composes
+  composes,
+  images
```

Comment:
What are your thoughts on combining `images` and `composes` into one, and having all things related to image creation together?

Reply:
Oh right, composes object has the list of the created images. We should really stick to either composes or images for naming vars, I know realized it's the same things.
Your comment makes sense. I will update accordingly.

Reply:
👍 BTW, I like "images" but am fine with either label.

Reply:
I would just move the previous imageTypes object into the existing composes object, so that I will not introduce huge change. But feel free to change it in future.